README.md

```diff
@@ -1,5 +1,5 @@
 [![Maven Central](https://img.shields.io/maven-central/v/io.github.rakutentech.inappmessaging/inappmessaging)](https://search.maven.org/artifact/io.github.rakutentech.inappmessaging/inappmessaging)
-[![CircleCI](https://circleci.com/gh/rakutentech/android-inappmessaging.svg?style=svg)](https://circleci.com/gh/rakutentech/android-inappmessaging)
+![Build status](https://app.bitrise.io/app/e9cc83da00ffd2b1/status.svg?token=4E3R-Baoxp0iPav7uiw1sA&branch=master)
```

Comment:
Is it safe to expose the account access token here? I wonder if GitHub encrypted secrets can be used in .md files?

Reply:
I think this token is only used to access the build status image and cannot be used for other purpose.
@donnie-jp Please advise if it is still ok to use this badge image in README.
Similarly, it is added in https://raw.githubusercontent.com/rakutentech/ios-inappmessaging/master/README.md as well.

Reply:
Yeah it seems like it's fine.
When you click the badge on a bitrise project it says the token is a "status image API token". It isn't a general purpose token. Also, on [here ](https://blog.bitrise.io/post/announcing-status-images) it says
> we created a new read-only token just for interacting with the status image, so you can show it anywhere and still have your project in full safety